src/swagger.json

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
Just out of curiosity, did you generate this with a tool, or did you hand-write it?

Reply:
A mixture.
I started by changing the sample yaml provided at https://editor.swagger.io/ to meet my implementation. That provided me the initial strucuture and the basis to understand what was going on.
When I had enough of a grasp on it, I exported a json from there and finished editing it in VSCode. I installed "OpenAPI (Swagger) Editor" as an extension to help me navigate that file better, but I done the edits by hand, to help me make sure of the steps I was talking.

Reply:
Very good!